predictor-server/src/main/java/ru/dbzukunft/predictor/server/service/datacrawler/AlphaVantageDataCrawlerService.java

```diff
@@ -0,0 +1,58 @@
+package ru.dbzukunft.predictor.server.service.datacrawler;
+
+import lombok.AllArgsConstructor;
+import org.slf4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.RestTemplate;
+import ru.dbzukunft.predictor.server.entity.AlphaVantageMetaData;
+import ru.dbzukunft.predictor.server.entity.AlphaVantageTimeSeries;
+import ru.dbzukunft.predictor.server.repository.AlphaVantageMetaDataRepository;
+import ru.dbzukunft.predictor.server.repository.AlphaVantageTimeSeriesRepository;
+import ru.dbzukunft.predictor.server.util.dto.AlphaVantageApiDTO;
+import ru.dbzukunft.predictor.server.util.exception.ServerException;
+import ru.dbzukunft.predictor.server.util.mapper.AlphaVantageMapper;
+
+import java.util.List;
+
+@Service
+@EnableScheduling
+@AllArgsConstructor
+public class AlphaVantageDataCrawlerService implements DataCrawlerService<AlphaVantageApiDTO> {
+
+    private Logger log;
+    private RestTemplate restTemplate;
+    private String alphaVantageUrl;
+    private AlphaVantageMetaDataRepository metaDataRepository;
+    private AlphaVantageTimeSeriesRepository timeSeriesRepository;
+
+    @Override
+    @Scheduled(fixedRate = 10000)
+    public void getDataFromApi() {
```

Comment:
I would add some debug statements before and after request to remove service. Could be useful to troubleshoot wrong config params/slow service response/etc. Like
```
log.debug("Requesting time series data using [{}] url", equityType, url);
// .. call service
log.debug("Successfully received time series data using [{}] url", equityType, url);
```

Reply:
Will add aspects